core/src/commonMain/kotlin/com/ustadmobile/core/impl/di/CommonDomainModule.kt

```diff
@@ -30,6 +35,10 @@ import org.kodein.di.singleton
  * at the Application level di.
  */
 fun commonDomainDiModule(endpointScope: EndpointScope) = DI.Module("CommonDomain") {
+
+    bind<ZipFileUseCase>() with singleton { createZipFileUseCase() }
```

Comment:
The DI binding for a UseCase which has different implementations on different platforms (or no implementation on certain platforms) should be done in platform specific DI - that is also now documented on the coding style guidelines. 

Please remove expect/actual for creation e.g. createZipFileUseCase etc.